packages/postgres-database/src/simcore_postgres_database/models/project_networks.py

```diff
@@ -0,0 +1,29 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+from .base import metadata
+from .projects import projects
+
+project_networks = sa.Table(
+    "project_networks",
+    metadata,
+    sa.Column(
+        "project_uuid",
+        sa.String,
+        sa.ForeignKey(
+            projects.c.uuid,
+            name="fk_project_networks_project_networks_projects",
+            ondelete="CASCADE",
+            onupdate="CASCADE",
+        ),
+        primary_key=True,
+        doc="project reference and primary key for this table",
+    ),
+    sa.Column(
+        "networks_with_aliases",
+        JSONB,
+        nullable=False,
+        server_default=sa.text("'{}'::jsonb"),
+        doc="Maps networks with attached services by node_id and alias",
```

Comment:
So this is something like `` networks_with_aliases = { 'some_network_name': { node1_id: "alias_node1", node2_id: "alias_node2" }, ... `` ?